project_timesheet_currency/models/analytic.py

```diff
@@ -22,12 +22,15 @@ def _get_timesheet_cost(self, values):
                 date=values['date']).compute(res['amount'], base_currency)
         return res
 
-    amount_currency = fields.Float(compute="_get_amount_currency", store=True)
+    amount_currency = fields.Float(
+        compute="_compute_get_amount_currency",
```

Comment:
It should be `_compute_amount_currency`

Reply:
thanks for the fast response. fix is pushed